wordpress-mobile/AztecEditor-iOS
# Remove gridicons references from the main Aztec pod.

Only keep references that are done on the demo project.

There is only things that I see as issues if we remove Gridicons from the pod:

 - We don't get a default Missing Image on the TextView -> but this could be set for however is using the pod
 - We don't get a play icon on the the Video attachment, I was considering implementing a default play button icon using some CoreGraphics code ( a triangle with round corners)

If we agree with this solution I can go ahead and implement the play icon and remove the Gridicon dependency.

To test:
 - Build the pod
 - Check if all tests build and pass
 - Build the example
 - Run the example and check if nothing breaks.
